chart/templates/secrets.yaml

```diff
@@ -22,7 +22,7 @@ stringData:
   POSTGRES_USER: {{ $user | quote }}
   POSTGRES_PASSWORD: {{ $password | quote }}
   POSTGRES_HOST: {{ $host | quote }}
-  DB_URL:  {{ (print "postgres://" $user ":" $password "@" $host "/incident_commander") | quote }}
+  DB_URL:  {{ (print "postgres://" $user ":" $password "@" $host "/incident_commander?sslmode=disable" ) | quote }}
```

Comment:
```suggestion
  DB_URL:  {{ (print "postgres://" $user ":" $password "@" $host "/incident_commander?sslmode=allow" ) | quote }}
```

Reply:
@moshloop Using `allow` will give you this error:
```
Failed to initialize the db: postgres: scanning system variables: pq: unsupported sslmode "allow"; only "require" (default), "verify-full", "verify-ca", and "disable" supported
```

golang's `lib/pq` does not support allow: https://github.com/lib/pq/blob/master/ssl.go#L15-L51